_includes/releases/v21.2/v21.2.10.md

```diff
@@ -0,0 +1,75 @@
+
+- Added a `changefeed.backfill.scan_request_size` [cluster setting](../v21.2/cluster-settings.html) to control scan request size during [backfill](../v21.2/use-changefeeds.html#schema-changes-with-column-backfill). [#79709][#79709]
+
+<h3 id="v21-2-10-sql-language-changes">SQL language changes</h3>
+
+- A `pgerror` with code `22P02` is now returned when an invalid cast to `OID` is made. [#79849][#79849]
+- An incremental [backup](../v21.2/backup.html) now fails if the `AS OF SYSTEM TIME` is less than the previous backup's end time. [#80287][#80287]
+
+<h3 id="v21-2-10-db-console-changes">DB Console changes</h3>
+
+- Filtering by column is added to [**Hot Ranges**](../v21.2/ui-hot-ranges-page.html) page. [#79645][#79645]
+- Added dropdown filter on the Node Diagnostics page to view by active, decommissioned, or all nodes. [#80336][#80336]
+
+<h3 id="v21-2-10-bug-fixes">Bug fixes</h3>
+
+- The execution time as reported on `DISTSQL` diagrams within the statement bundle collected via [`EXPLAIN ANALYZE (DEBUG)`](../v21.2/explain-analyze.html#explain-analyze-debug) no longer is negative when the statement encountered an error. [#79369][#79369]
+- An internal error when the inner expression of a column access expression evaluated to `NULL` no longer occurs. For example, evaluation of the expression `(CASE WHEN b THEN ((ROW(1) AS a)) ELSE NULL END).a` would error when `b` is `false`. [#79528][#79528]
+- An error when accessing a named column of a labelled tuple no longer occurs. The error occurred when an expression could produce one of several different tuples. For example, `(CASE WHEN true THEN (ROW(1) AS a) ELSE (ROW(2) AS a) END).a` would fail to evaluate. [#79528][#79528]
+- [Pebble](../v21.2/architecture/storage-layer.html#pebble) compaction heuristics no longer allow a large compaction backlog to accumulate, eventually triggering high read amplfiication. [#79611][#79611]
+- HTTP 304 responses no longer result in error logs. [#79860][#79860]
+- A custom time series metric `sql.distsql.queries.spilled` is no longer computed incorrectly leading to exaggerated number. [#79881][#79881]
+- [`nextval` and `setval`](../v21.2/create-sequence.html#sequence-functions) are non-transactional except when it is called in the same transaction that the sequence was created in. Creating a sequence and calling `nextval` pr `setval` on it within a transaction no longer causes the query containing `nextval` to hang. [#79866][#79866]
+- The [SQL Activity](../v21.2/ui-overview.html#sql-activity) page no longer returns "descriptor not found" error during v21.1-v21.2 mixed version state. [#79795][#79795]
+- [Resetting SQL statistics](../v21.2/ui-statements-page.html#statement-statistics) in v21.1-v21.2 mixed version state no longer causes "descriptor not found" error. [#79795][#79795]
+- CockroachDB no longer attempts to flush statistics into disk during v21.1-v21.2 mixed version state and log "descriptor not found" error messages. [#79795][#79795]
```

Comment:
suggest more common "flush statistics to disk"

this is hard to understand as phrased.  suggest reordering the clauses, something like

"During v21.1-v21.2 mixed version state, CockroachDB no longer attempts to flush stats to disk.  It also does not log ... error messages" 

or something like that